int64 time_s
geometry_msgs/Pose start_pose
geometry_msgs/Twist start_twist
---
geometry_msgs/Pose predicted_pose